packages/snaps-simulation/src/helpers.ts

```diff
@@ -126,6 +126,20 @@ export type SnapHelpers = {
     keyringRequest: KeyringOptions,
   ): Promise<SnapResponseWithoutInterface>;
 
+  /**
+   * Get the response from the Snap's `onInstall` handler.
+   *
+   * @returns The response.
+   */
+  onInstall(): Promise<SnapResponseWithInterface>;
```

Comment:
We should probably let developers pass the `origin` for these handlers. We can add a request object as with the others so its easy to add more params later

Reply:
Added optional `(request?: RequestOptions)`.